### ISSUE TITLE
NPM: dependencies missing?

### ISSUE THREAD

**Issue by antoniobrandao** [everyonesdesign/cascade#4]
Just got this after a fresh "npm install cascade-layout":

```
[BS] File changed: builds/development/resources/js/bundle.js
[00:45:23] gulp-notify: [[handleError.js] Compile Error] Error: Cannot find module 'xtend/mutable' from '/Users/antoniobrandao/WEBROOT/vvvpt/node_modules/cascade-layout'
[00:45:23] Finished 'bundle' in 1.99 h
[00:45:23] gulp-notify: [[handleError.js] Compile Error] Error: Cannot find module 'mucss/css' from '/Users/antoniobrandao/WEBROOT/vvvpt/node_modules/cascade-layout'
[00:45:23] gulp-notify: [[handleError.js] Compile Error] Error: Cannot find module 'mucss/margins' from '/Users/antoniobrandao/WEBROOT/vvvpt/node_modules/cascade-layout'
```

Running on Browserify 8.1.3 / Gulp 3.8.10 / Yosemite

**Comment by antoniobrandao** [comment]
Checked, and the dependecies "ccjs", "emmy", "mucss", "queried", "xtend" are not being installed. Not sure but I think this can be solved by moving the "devDependencies" in your package.json into "dependencies" and re-publishing.

**Comment by everyonesdesign** [comment]
@antoniobrandao 
Right, thank you very much for submiting this issue.
Solved in 17b490645e0c8430dcf25b48115fe55da827f1a7

**Comment by antoniobrandao** [comment]
thanks !
